packages/base/core/src/index.ts

```diff
@@ -33,4 +33,6 @@ export { default as EventEmitter } from './utils/EventEmitter'
 
 export { default as loadDeveloperTools } from './devtools'
 
-loadDeveloperTools()
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Please change this to `process.env.NODE_ENV !== 'production` as it not guaranteed that the `NODE_ENV` is set at all.
Also add `/* istanbul ignore if -- @preserve */` direct before this line. This will turn off the coverage check, as it is not needed in this case